airflow/providers/google/cloud/operators/datapipeline.py

```diff
@@ -74,34 +74,27 @@ def execute(self, context: Context):
         )
 
         # returns the full response body
-        return self.data_pipeline
+        return self.data_pipeline["name"]
```

Comment:
Return the full pipeline instead of just the name.

Reply:
updated